programs/address-lookup-table-tests/tests/common.rs

```diff
@@ -0,0 +1,107 @@
+use {
+    solana_address_lookup_table_program::{
+        id,
+        processor::process_instruction,
+        state::{AddressLookupTable, LookupTableMeta},
+    },
+    solana_program_test::*,
+    solana_sdk::{
+        account::AccountSharedData,
+        clock::Slot,
+        hash::Hash,
+        instruction::Instruction,
+        instruction::InstructionError,
+        pubkey::Pubkey,
+        signature::{Keypair, Signer},
+        slot_hashes::SlotHashes,
+        transaction::{Transaction, TransactionError},
+    },
+    std::borrow::Cow,
+};
+
+#[allow(dead_code)]
+pub async fn setup_test_context() -> ProgramTestContext {
+    let program_test = ProgramTest::new("", id(), Some(process_instruction));
```

Comment:
interesting, so "native mode" in program-test works for native programs. good to know!

Reply:
Indeed it does! There could be some weird edge cases still but I think we can officially move away from all the nasty mock stuff that's used for testing the vote program, for example.